scripts/populate_conferences.py

```diff
@@ -505,6 +495,27 @@ def main():
         'poster': True,
         'talk': True,
     },
+    'PsiChiRepository': {
+        'name': 'Psi Chi',
+        'info_url': 'http://psichi.org',
+        'logo_url': None,
+        'active': True,
+        'admins': [],
+        'public_projects': True,
+        'poster': True,
+        'talk': True,
+        'field_names': {
+            'submission1': 'measure',
+            'submission2': 'material',
```

Comment:
One suggestion for future improvement would be to allow for more than 2 submission types:

```python
'field_names': [
    ('measure', 'measures/scales'),
    ('materials', 'study materials'),
    ('data', 'data'),
]